imbi/endpoints/project_ssm_configuration.py

```diff
@@ -96,11 +96,15 @@ async def get(self, *args, **kwargs) -> None:
 
         output = [{
             'name': name,
-            'values': [{
-                'environment': environment,
-                'value': data['value'],
-                'type': data['type']
-            } for environment, data in d.items()]
+            'values': {
+                environment: {
+                    'value': data['value'],
+                    'type': data['type']
+                }
+                for environment, data in d.items()
+            },
+            'self': self.request.full_url() + '/' +
+            urllib.parse.quote(name, safe=[])
```

Comment:
You should be able to use `self.reverse_url('project-configuration', name)` here instead. I think that will result in a relative reference which _should_ be okay as long as the UI is fetching it correctly.